### PR TITLE
Adding AstroGlue recipe

### PR DESCRIPTION
**Pull request by Kam-s18**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Kam-s18** [comment]
@conda-forge/staged-recipes @conda-forge/help-python

Ready for review. Thanks in advance!

File: recipes/AstroGlue/yum_requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+xxorg-x11-server-Xvfb
+gtk2-devel
```
Comment (xhochy) [inline recipes/AstroGlue/yum_requirements.txt:2]: Why is this needed? We have this as a conda package.

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/AstroGlue/yum_requirements.txt (hunk 1 above)
Comment (xhochy) [reply recipes/AstroGlue/yum_requirements.txt:2]: This should be manageable without the `gtk2-devel` line.

Comment (Kam-s18) [reply recipes/AstroGlue/yum_requirements.txt:2]: Thank you! 
I will remove `gtk2-devel` line, re-run the pipelines and add a mention for review once that is done.

Comment (Kam-s18) [reply recipes/AstroGlue/yum_requirements.txt:2]: In the absence of the `gkt2-devel` line in the `yum_requirements.txt`, pipelines testing for Windows environment fails with the following error:
image

All the additional requirements seem to be necessary for a successful run. I added these requirements upon reviewing the glue-qt feedstock (which is also used in AstroGlue) - https://github.com/conda-forge/glue-qt-feedstock/blob/main/recipe/yum_requirements.txt

Please do let me know if there is any other way to resolve this.
Thanks in advance!

Comment (Kam-s18) [reply recipes/AstroGlue/yum_requirements.txt:2]: @xhochy 
Hi, I just wanted to check if there is anything else that I can do to get this resolved and re-request a review. The requirements files still includes `gkt2-devel` line as the Windows environment testing fails without it. 

Thank you in advance!

**Comment by Kam-s18** [comment]
@conda-forge/staged-recipes @conda-forge/help-python

The requested changes couldn't be accommodated as the additional requirements were necessary for passing the OS checks.
Ready for re-review. Thanks in advance!

File: recipes/AstroGlue/yum_requirements.txt (hunk 1 above)
Comment (xhochy) [reply recipes/AstroGlue/yum_requirements.txt:2]: `yum_requirements.txt` shouldn't have influence on Windows tests.

Comment (Kam-s18) [reply recipes/AstroGlue/yum_requirements.txt]: Hello!
I removed the `gkt2-devel` line from the `yum_requirements` file. All the pipelines tests have run successfully now. I apologise for the confusion earlier. Please do let me know if there's anything left to be done by me. Thank you so much!

**Review by xhochy — APPROVED** [review]
